.github/workflows/ci.yaml

```diff
@@ -31,7 +31,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - run: rustup toolchain install $RUST_VERSION --profile minimal
+      - run: |
+          rustup toolchain add nightly-2025-10-20 --profile minimal
```

Comment:
Now that we have a `rust-toolchain.toml` it's enough just to have `rustup toolchain install --profile minimal` here and it automatically gets the version from the file. And we don't need the stable toolchain either right?